teeworlds7/send_message_hooks.go

```diff
@@ -19,306 +19,307 @@ func userSendMsgCallback[T any](userCallbacks []func(T) bool, msg T) bool {
 func (client *Client) registerMessagesCallbacks(messages []messages7.NetMessage) []messages7.NetMessage {
 	filteredMessages := make([]messages7.NetMessage, 0, len(messages))
 
+outer:
 	for _, msg := range messages {
 		for _, callback := range client.Callbacks.MessageOut {
-			if callback(&msg) == false {
-				continue
+			if !callback(&msg) {
+				continue outer
 			}
 		}
 
 		switch castMsg := msg.(type) {
 		case *messages7.CtrlKeepAlive:
-			if userSendMsgCallback(client.Callbacks.CtrlKeepAliveOut, castMsg) == false {
+			if !userSendMsgCallback(client.Callbacks.CtrlKeepAliveOut, castMsg) {
```

Comment:
Stuff like that should be on its own pr. I typed out all the ``== false`` because I thought its more the go vibe. But I am totally fine with the ``!`` I use that everywhere else.

You can leave it in for now. I just wanted to say that a huge opinionated wall of style diff does not belong into a error propagation pr. Makes it harder to scroll through the diff and discuss what is relevant.

Reply:
` == true/false` is not Go style.
We got a linter for stuff like this, which is why I'm conforming to "staticcheck" warnings.
You can easily install it if you got the vscode Go extension
<img width="1816" alt="Bildschirmfoto 2024-09-12 um 18 10 07" src="https://github.com/user-attachments/assets/0c27b090-2abf-4d78-bcf9-fd1d57033219">
<img width="1816" alt="Bildschirmfoto 2024-09-12 um 18 12 16" src="https://github.com/user-attachments/assets/f99f7b88-8865-49f2-adca-b25a7d75a833">

Reply:
There is no style discussion in Go, as most of the expected styling is documented on the Go website. These discussions are a waste of time.

Reply:
Ah nice to hear that a linter agrees.